aio/deploy/helm-chart/kubernetes-dashboard/Chart.yaml

```diff
@@ -15,7 +15,7 @@
 apiVersion: v1
 name: kubernetes-dashboard
 version: 2.0.1
-appVersion: 2.0.1
+appVersion: 2.0.2
```

Comment:
Oh no... the chart version wasn't changed, so I can get it with `helm dep up`, it takes old 2.0.1 version all along.

Reply:
@floreks You have to bump chart version with each `appVersion` bump to propagate it. Otherwise Helm will look to local cache instead. Can you bump it, please?

Reply:
Done

Reply:
@floreks please, re-run release CI job for this also, the version is missing [in repo index](https://kubernetes.github.io/dashboard/index.yaml) at the moment.

Reply:
I will do the new release today.

Reply:
For the moment, release is manual. We are in the process of setting up proper CI (like prevent bad versions like this), then automate CD, which should be operational in a few days. In the meantime, not all charts version will be published.